extensions/amp-form/0.1/amp-form.js

```diff
@@ -192,7 +197,10 @@ function installSubmissionHandlers(win) {
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        installSubmissionHandlers(win);
+        actionServiceFor(win).addEvent('submit');
```

Comment:
If you do add `submit-success` and `-error` - you'll probably have to trigger them manually via `actionServiceFor(window).trigger(this.form, 'submit-success', null);` or such, since there's no standard events to map to.